TCAT/InformationViewController.swift

```diff
@@ -28,6 +28,10 @@ class InformationViewController: UIViewController {
         
         title = "About Us"
         
+        let scrollView = UIScrollView(frame: view.frame)
+        scrollView.contentSize = view.frame.size
+        view = scrollView
+        
```

Comment:
See what you're doing but think it would read better if you add `scrollView` to the view and then add all the other views using `scrollView.addSubview()`